examples/clone/clone.description

```diff
@@ -1,6 +1,6 @@
-// Clone built-in method is used to make a copy of any anydata value.
+// The `.clone()` built-in method is used to make a copy of an `anydata` value.
```

Comment:
Can we add why we would want to make a copy here? That use case type information is missing in this example.

Reply:
IMO, in this case the use case IS making a copy. 

Will re-visit this and the other examples in the next iteration to see if there is any way we can incorporate more information about possible usage.